wolfcrypt/src/evp.c

```diff
@@ -792,6 +800,68 @@ static int wolfSSL_EVP_CipherUpdate_CCM(WOLFSSL_EVP_CIPHER_CTX *ctx,
 }
 #endif /* HAVE_AESCCM */
 
+#if defined(HAVE_ARIA)
+static int wolfSSL_EVP_CipherUpdate_AriaGCM_AAD(WOLFSSL_EVP_CIPHER_CTX *ctx,
+        const unsigned char *in, int inl) {
+    if (in && inl > 0) {
+        byte* tmp = (byte*)XREALLOC(ctx->authIn,
```

Comment:
`XREALLOC` is not portable. Recommend a support where `XREALLOC` is not defined. We have a few examples of this in the code. Is `DYNAMIC_TYPE_OPENSSL` the right type?

Reply:
[AESGCM](https://github.com/bandi13/wolfssl/blob/ARIA-cipher/wolfcrypt/src/evp.c#L657) and [AESCCM](https://github.com/bandi13/wolfssl/blob/ARIA-cipher/wolfcrypt/src/evp.c#L750) function behave similarly.